configure.ac

```diff
@@ -113,6 +113,13 @@ AC_CHECK_FUNC([pthread_spin_init],
 	[have_spinlock=1],
 	[have_spinlock=0])
 
+if test "$macos" == "0"; then
```

Comment:
I think that should be either `-eq` or `=` (single equal sign, not double).  A comment explaining why it doesn't need to be checked on Mac is probably also warranted.